pages/library/_viewId.vue

```diff
@@ -1,36 +1,34 @@
 <template>
   <div>
-    <h1>{{ Name }}</h1>
-    <v-list>
-      <v-list-item
+    <h1>
+      <span>{{ Name }}</span>
+    </h1>
+    <div class="cardsContainer">
+      <CardBuilder
```

Comment:
Prefer names like `card-builder` to stay in sync with the rest of the tags.

I'll look if there's a kebab-case rule for component tags somewhere :smiley:

Reply:
I've added it in #39